tests/test_job.py

```diff
@@ -500,3 +500,22 @@ def test_job_resubmit_all(hq_env: HqEnv):
 
     table = hq_env.command(["resubmit", "1"], as_table=True)
     assert table[3] == ["Tasks", "3; Ids: 2, 7, 9"]
+
+def test_job_tasks_table(hq_env:HqEnv):
+    hq_env.start_server()
+    hq_env.command(["submit", "echo", "test"])
+    table = hq_env.command(["job", "1" , "--tasks"],as_table=True)
+    assert table[11][1] == "WAITING"
+    assert table[11][2] == ""
+    hq_env.start_worker()
+    table = hq_env.command(["job", "1" , "--tasks"],as_table=True)
```

Comment:
Use `wait_for_job_state` to make sure that the test will not fail if the task execution is too slow.